src/app/component/general/eco-news/news-list/news-list.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="wrapper">
+  <app-remaining-count [remainingCount]="remaining"></app-remaining-count>
+  <app-change-view-button (view)="chageView($event)"></app-change-view-button>
+</div>
+
+<div>
+  <ul class="list" [ngClass]="{ 'gallery-view-active': view }">
+    <li [ngClass]="{
+        'gallery-view-li-active': view,
+        'list-view-li-active': !view
+      }" *ngFor="let data of elements|filterEcoNews:gridOutput">
+      <app-news-list-gallery-view *ngIf="view" [ecoNewsModel]="data">
+      </app-news-list-gallery-view>
+      <app-news-list-list-view *ngIf="!view" [ecoNewsModel]="data">
+      </app-news-list-list-view>
+    </li>
+  </ul>
+</div>
+
+<div infiniteScroll [infiniteScrollDistance]="0" [infiniteScrollThrottle]="500" (scrolled)="onScroll()"></div>
```

Comment:
If there are more then 2 attributes replace it to one column

Reply:
Fixed